src/main/java/uk/org/tombolo/importer/londondatastore/LondonPHOFImporter.java

```diff
@@ -21,9 +21,23 @@
  * Importer for the Public Health Outcomes Framework (PHOF) indicators for London.
  *
  * URL: http://data.london.gov.uk/dataset/public-health-outcomes-framework-indicators
+ * FILE : https://files.datapress.com/london/dataset/public-health-outcomes-framework-indicators/2015-11-10T12:05:53
+ * /phof-indicators-data-london-borough.xlsx
  *
+ */
+
+/**
+ * INFO FOR RECIPE
  *
+ * "importerClass": "uk.org.tombolo.importer.londondatastore.LondonPHOFImporter"
+ * "datasourceId": "phofIndicatorsLondonBorough"
+ * "provider": "uk.gov.london"
+ * "subjectTypes": [localAuthority]
+ *
+ * "timedValueAttributes": [ See file]
+ * "fixedValueAttributes": []
```

Comment:
which file?

Reply:
the datasource file in line 24. I will change it to catalogue.json to be consistent anyway.